doc/deployment/deploy_custom.md

```diff
@@ -84,203 +43,219 @@ pachctl deploy custom --persistent-disk <persistent disk backend> --object-store
 It is likely that your on-premises Kubernetes installation is configured to use StatefulSets.
 
-The `--dynamic-etcd-nodes` flag has a parameter which specifies the number of `etcd` nodes which your deployment will create.
-Pachyderm recommends you keep this number at 1.
-Consult with your Pachyderm support team if you want to change it.
+The `--dynamic-etcd-nodes` flag specifies the number of `etcd` nodes that your deployment creates.
+Pachyderm recommends that you keep this number at `1`.
+If you want to change it, consult with your Pachyderm support team.
+
+This flag creates a `VolumeClaimTemplate` in the `etcd` `StatefulSet` that uses the standard `etcd-storage-class`.
 
-This flag will create a `VolumeClaimTemplate` in the `etcd` `StatefulSet` that uses the standard `etcd-storage-class`.
+**NOTE** Consult with your Kubernetes administrator about the StorageClass 
+that you should use for `etcd` in your Kubernetes deployment.
+If you need to use a different one than the default setting, 
+you can use the `--etcd-storage-class` flag to specify the StorageClass.
 
-**NOTE** Consult with your Kubernetes administrator on the StorageClass you should use for `etcd` in your Kubernetes deployment.
-If it's not the default, you can use the flag `--etcd-storage-class` to specify the StorageClass.
 
 `--static-etc-volume` is used when your Kubernetes installation has not been configured to use StatefulSets.
-It will use a static volume with Pachyderm's `etcd`, 
-creating a PV with a spec appropriate for each of cloud providers specified above:
+When you specify `--static-etc-volume` flag, Pachyderm creates a static volume for `etcd`. 
+Pachyderm creates a PV with a spec appropriate for each of the cloud providers:
 
 - `aws`: awsElasticBlockStore for Amazon Web Services
 - `azure`: azureDisk for Microsoft Azure
 - `google`: gcePersistentDisk for Google Cloud Storage
 
-Of course, 
-these choices are not relevant for most on-premises deployments,
-so you will need manually edit your manifest 
-after consulting with your Kubernetes administrators
-to determine the correct choices for your infrastructure.
+These choices are not applicable to  most on-premises deployments.
+To determine the correct choices for your on-prem infrastructure, 
+consult with your Kubernetes administrators
+and edit your manifest manually.
 
-The section on that [storage manifests](#persistentvolume) goes into a little more detail on this.
+For more information, see [storage manifests](#persistentvolume).
 
 #### Persistent disk parameters
 
 Regardless whether you choose to deploy with StatefulSets or static volumes,
 the `--persistent-disk` flag takes two arguments
 that you specify right after the single argument to the `--object-store` flag.
 
-[The first argument is always ignored,
-but must be present.](https://github.com/pachyderm/pachyderm/issues/3312)
-You may set it to any text value you like.
+Although the first argument is required, Pachyderm ignores it.
```

Comment:
Later one you say:

> Note: we use any-string as the first argument above because, while the deploy custom command expects 6 arguments, it will ignore the first argument when deploying with StatefulSets.

If this is the technical requirement for why this exists, i can be make much clearer as you do in the note later.